policy/policy_handler/mocks/base_image_details.go

```diff
@@ -0,0 +1,116 @@
+package mocks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/atomist-skills/go-skill"
+	"github.com/atomist-skills/go-skill/policy/data"
+	"github.com/atomist-skills/go-skill/policy/types"
+)
+
+const (
+	ImageDetailsQueryName = "image-details-by-digest"
+
+	baseImagesByDigestQuery = `
+	query ($context: Context!, $digest: String!) {
+		baseImagesByDigest(context: $context, digest: $digest) {
+		  images {
+			digest
+			tags {
+			  current
+			  name
+			}
+			repository {
+				hostName
+				repoName
+			}
+		  }
+		}
+	}`
+)
+
+type (
+	GqlImagePlatform struct {
+		Architecture string `json:"architecture"`
+		Os           string `json:"os"`
+		Variant      string `json:"variant,omitempty"`
+	}
+
+	Repository struct {
+		HostName string `json:"hostName" edn:"hostName"`
+		RepoName string `json:"repoName" edn:"repoName"`
+	}
+
+	Tag struct {
+		Name    string `json:"name" edn:"name"`
+		Current bool   `json:"current" edn:"current"`
+	}
+
+	BaseImage struct {
+		Digest     string     `json:"digest" edn:"digest"`
+		Repository Repository `json:"repository" edn:"repository"`
+		Tags       []Tag      `json:"tags" edn:"tags"`
+	}
+
+	ImageDetailsByDigest struct {
+		Digest       string     `json:"digest" edn:"digest"`
+		BaseImage    *BaseImage `json:"baseImage" edn:"baseImage"`
+		BaseImageTag *Tag       `json:"baseImageTag" edn:"baseImageTag"`
+	}
+
+	ImageDetailsByDigestResponse struct {
+		ImageDetailsByDigest *ImageDetailsByDigest `json:"imageDetailsByDigest" edn:"imageDetailsByDigest"`
+	}
+
+	BaseImagesByDigest struct {
+		Images []BaseImage `json:"images" edn:"images"`
+	}
+
+	BaseImagesByDigestResponse struct {
+		BaseImagesByDigest *BaseImagesByDigest `json:"baseImagesByDigest" edn:"baseImagesByDigest"`
+	}
+)
+
+func MockBaseImageDetails(ctx context.Context, req skill.RequestContext, sb *types.SBOM) (ImageDetailsByDigestResponse, error) {
+	ds, err := data.NewSyncGraphqlDataSource(ctx, req)
+	if err != nil {
+		return ImageDetailsByDigestResponse{}, err
+	}
+
+	baseImageDigest := sb.Source.Provenance.BaseImage.Digest
+
+	var queryResponse BaseImagesByDigestResponse
+
+	queryVariables := map[string]interface{}{"digest": baseImageDigest}
+	_, err = ds.Query(ctx, "base-images-by-digest", baseImagesByDigestQuery, queryVariables, &queryResponse)
```

Comment:
Shouldn't this be `image-details-by-digest` which is the value of the const `ImageDetailsQueryName`?

```suggestion
	_, err = ds.Query(ctx, ImageDetailsQueryName, baseImagesByDigestQuery, queryVariables, &queryResponse)
```

Reply:
No, this is a different query. `ImageDetailsQueryName` is what we are mocking, `"base-images-by-digest"` is a separate query we need to make to enrich the data in the sbom in order to do the mocking.